rabbitStairs/Program.cs

```diff
@@ -37,55 +36,31 @@ public static void Main()
                 
             }
 
-            // Checking user input and fix handling errors
-            if (lineObj.k < 1)
-            {
-                Console.WriteLine("Quantity of jumps can't be less than 1. Program will be close.");
-                // Delay
-                Console.ReadLine();
-                Environment.Exit(-1);
-            }
-
-            if (lineObj.n < lineObj.k)
-            {
-                Console.WriteLine("Quantity of stairs can't be less than quantity of jumps. Program will be close.");
-                // Delay
-                Console.ReadLine();
-                Environment.Exit(-1);
-            }
-
-            if (lineObj.n > 300)
-            {
-                Console.WriteLine("The stair is too high for rabbit. It can't be grater than 300 steps. Program will be close.");
-                // Delay
-                Console.ReadLine();
-                Environment.Exit(-1);
-            }
-
-
-            result = lineObj.Rabbit(lineObj.k, lineObj.n);
+            lineObj.CheckInput(lineObj.Steps, lineObj.QtyStairs);
+          
```

Comment:
Write some statement here like steps ??? QtyStairs ??? start calculating....
Because I do not know if program hung or works and if works if it parsed my input correctly